syntax_error_note_php70.md

```diff
@@ -1,29 +1,13 @@
-Syntax error found in 7 files
+Syntax error found in 4 files
 
 ------------------------------------------------------------
-Parse error: rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Input/StringInput.php:25
-    23| class StringInput extends \RectorPrefix20210516\_HumbugBox0b2f2d5c77b8\Symfony\Component\Console\Input\ArgvInput
-    24| {
-  > 25|     public const REGEX_STRING = '([^\\s]+?)(?:\\s|(?<!\\\\)"|(?<!\\\\)\'|$)';
-    26|     public const REGEX_QUOTED_STRING = '(?:"([^"\\\\]*(?:\\\\.[^"\\\\]*)*)"|\'([^\'\\\\]*(?:\\\\.[^\'\\\\]*)*)\')';
-    27|     /**
-Syntax error in rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Input/StringInput.php on line 25
-------------------------------------------------------------
-Parse error: rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Output/Output.php:37
-    35|      * @param OutputFormatterInterface|null $formatter Output formatter instance (null to use default OutputFormatter)
-    36|      */
-  > 37|     public function __construct(?int $verbosity = self::VERBOSITY_NORMAL, bool $decorated = \false, \RectorPrefix20210516\_HumbugBox0b2f2d5c77b8\Symfony\Component\Console\Formatter\OutputFormatterInterface $formatter = null)
-    38|     {
-    39|         $this->verbosity = null === $verbosity ? self::VERBOSITY_NORMAL : $verbosity;
-Syntax error in rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Output/Output.php on line 37
-------------------------------------------------------------
-Parse error: rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Formatter/OutputFormatter.php:173
-    171|      * Tries to create new style instance from string.
-    172|      */
-  > 173|     private function createStyleFromString(string $string) : ?\RectorPrefix20210516\_HumbugBox0b2f2d5c77b8\Symfony\Component\Console\Formatter\OutputFormatterStyleInterface
-    174|     {
-    175|         if (isset($this->styles[$string])) {
-Syntax error in rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Formatter/OutputFormatter.php on line 173
+Parse error: rector-prefixed-downgraded-php70/vendor/phpstan/phpstan-extracted/vendor/symfony/console/Event/ConsoleCommandEvent.php:25
+    23|      * The return code for skipped commands, this will also be passed into the terminate event.
+    24|      */
+  > 25|     public const RETURN_CODE_DISABLED = 113;
```

Comment:
Maybe consider increating the print limit to 40 seconds.
There is much more files to print

Reply:
ok, increased to 40 https://github.com/rectorphp/rector-src/pull/50/commits/e9f4601252af7b4f7654df85d65a37da1cb28118